tools/convert_to_ir.py

```diff
@@ -14,16 +14,21 @@
 # See the License for the specific language governing permissions
 # and limitations under the License.
 
-import argparse
+import os
```

Comment:
why did you sort it this way?

Reply:
1. Standard library imports.
1. Related third party imports.
1. Local application/library specific imports.